_posts/0000-01-02-Wadzachitie.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Almost there. One day when I understand everything I will look back and laugh at myself.

Reply:
Cool